packages/back-end/src/services/experiments.ts

```diff
@@ -1930,19 +1951,15 @@ export function updateExperimentApiPayloadToInterface(
   };
 }
 
-export async function getRegressionAdjustmentInfo(
+export async function getSettingsForSnapshotMetrics(
   context: ReqContext | ApiReqContext,
   experiment: ExperimentInterface
 ): Promise<{
   regressionAdjustmentEnabled: boolean;
-  metricRegressionAdjustmentStatuses: MetricRegressionAdjustmentStatus[];
+  settingsForSnapshotMetrics: MetricSnapshotSettings[];
 }> {
   let regressionAdjustmentEnabled = false;
-  const metricRegressionAdjustmentStatuses: MetricRegressionAdjustmentStatus[] = [];
-
-  if (!experiment.regressionAdjustmentEnabled) {
-    return { regressionAdjustmentEnabled, metricRegressionAdjustmentStatuses };
```

Comment:
This change means when RA is disabled for an experiment, it will now return a non empty array of metric regression statuses. Probably fine, but just wanted to call out in case it wasn't intentional

Reply:
Yeah I didn't see any issues in testing as this flag gets used again later when building the actual array of settings. I'll check more carefully though.